package.json

```diff
@@ -639,6 +643,21 @@
               "providerUrl": "http://127.0.0.1:8545",
               "disableFetchExternal": false
             }
+          },
+          {
+            "label": "Truffle - Debugger Launch using config network",
+            "description": "Runs the Truffle debugger (truffle) and attaches to a TestRPC instance",
```

Comment:
Super minor thing but just wondering if we should call it a "Ganache" instance vs "TestRPC" (given the latter is super dated)

Reply:
Thanks @kevinbluer for catching this. I've changed it to _Ganache_. Before I used _TestRPC_ because it was used in a similar snippet, so I changed both.